### DISCUSSION TITLE
Cards: Explicitly state "player(s)"

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#17] [Ideas]
So I started going through and change "US" to "US player", "Russia" to "Russian player", but it's turns out quite noisy. Examples:

> West players lose Supremacy over all Economy Slots.

> **Played by Russian player**: Russian player loses ..

> **Permanent effect**: US player cannot conduct Influence Ops in Cuba.

> US player conducts a Destabilization .. If successful, US player repeats ..

Perhaps it's better to just add it where it might be unclear? I'm going to go through the card to see where it might be prudent.

**Comment by rodrigoSantamaria** [comment]
I'm for word economy too. I think in most of the cases you don't need the 'player' addenda, as most of the time "Russia" refers to players, it's the subject, and when it's the country, it's the object of the sentence. So I agree, only when there might be confusion.

**Comment by RebelusQuo** [comment]
I reviewed the discussion where we first mentioned this and it had to do with phrases where the original text started with mentioning a superpower followed by the instruction to either add or remove Influence. This is quite different from the cards I looked at, and I think ultimately this has to be verified by a full reading of my first draft.

Now, returning to the kind of phrasing I looked at here, it makes sense for West/East: "West player", "West players". However, when we talk about the individual superpowers the grammar begs for using "the" which makes it longer with little gain: "US" vs "the US player". In this spirit I am now finding card texts where we could *remove* "player":

Oil thirst: Here I had added "player" in my suggestion, so reverting back to
> US must conduct a ..

Kim Jong Il: Here I actually add "players" because it affects "West"
> *Permanent effect*: West players cannot ..

Cards with "prevent effects": With these effects the text reads better to add "player", but I'm also prefixing with "the", as discussed above
> .. , the Russian player can cancel ..

Madrid Bombings: Since this choice is done by a specific player I reworked it into a condition-like phrasing. Here I feel it reads better with "player", but to be consistent it should then start with "The". however, can we see past that when it starts the paragraph in a condition like this?
> **EU player chooses**: ..

**Reply by rodrigoSantamaria** [reply]
Honestly, I trust you in whatever you feel like the most clear way of writing it. I might be biased by overknowing the cards, so to me all of the options sound good, even the more 'unclear' one of using always only the power name, either as subject (player) or object (country).

**Reply by RebelusQuo** [reply]
Very well 👍
